scripts/foundry/mainnet/MainnetConstants.s.sol

```diff
@@ -5,22 +5,47 @@ import "../../../contracts/external/ProxyAdmin.sol";
 import "../../../contracts/external/TransparentUpgradeableProxy.sol";
 
 contract MainnetConstants {
-    address public constant GOVERNOR = 0xdC4e6DFe07EFCa50a197DF15D9200883eF4Eb1c8;
-    address public constant GUARDIAN = 0x0C2553e4B9dFA9f83b1A6D3EAB96c4bAaB42d430;
-    address public constant PROXY_ADMIN = 0x1D941EF0D3Bba4ad67DBfBCeE5262F4CEE53A32b;
-    address public constant PROXY_ADMIN_GUARDIAN = 0xD9F1A8e00b0EEbeDddd9aFEaB55019D55fcec017;
-    address public constant CORE_BORROW = 0x5bc6BEf80DA563EBf6Df6D6913513fa9A7ec89BE;
+    address constant GOVERNOR = 0xdC4e6DFe07EFCa50a197DF15D9200883eF4Eb1c8;
+    address constant GUARDIAN = 0x0C2553e4B9dFA9f83b1A6D3EAB96c4bAaB42d430;
+    address constant PROXY_ADMIN = 0x1D941EF0D3Bba4ad67DBfBCeE5262F4CEE53A32b;
+    address constant PROXY_ADMIN_GUARDIAN = 0xD9F1A8e00b0EEbeDddd9aFEaB55019D55fcec017;
+    address constant CORE_BORROW = 0x5bc6BEf80DA563EBf6Df6D6913513fa9A7ec89BE;
 
-    address public constant ANGLE_ROUTER = 0x4579709627CA36BCe92f51ac975746f431890930;
-    address public constant ONE_INCH = 0x1111111254EEB25477B68fb85Ed929f73A960582;
-    address public constant UNI_V3_ROUTER = 0xE592427A0AEce92De3Edee1F18E0157C05861564;
+    address constant ANGLE_ROUTER = 0x4579709627CA36BCe92f51ac975746f431890930;
+    address constant ONE_INCH = 0x1111111254EEB25477B68fb85Ed929f73A960582;
+    address constant UNI_V3_ROUTER = 0xE592427A0AEce92De3Edee1F18E0157C05861564;
 
     // AGEUR Mainnet treasury
-    address public constant AGEUR_TREASURY = 0x8667DBEBf68B0BFa6Db54f550f41Be16c4067d60;
-    address public constant AGGOLD_TREASURY = address(0);
+    address constant AGEUR_TREASURY = 0x8667DBEBf68B0BFa6Db54f550f41Be16c4067d60;
+    address constant AGGOLD_TREASURY = address(0);
 
-    uint256 public constant BASE_TOKENS = 10 ** 18;
-    uint64 public constant BASE_PARAMS = 10 ** 9;
+    uint256 constant BASE_TOKENS = 10 ** 18;
+    uint64 constant BASE_PARAMS = 10 ** 9;
+
+    /*//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+                                                      MORPHO DATA                                                   
+    //////////////////////////////////////////////////////////////////////////////////////////////////////////////////*/
+
+    address constant USDA = 0x0000206329b97DB379d5E1Bf586BbDB969C63274;
```

Comment:
should be fetched with the `_chainToContract` call

Reply:
Not setup on this repo and never managed to make it work